src/index.html

```diff
@@ -11,7 +11,241 @@
+            id="email"
+            placeholder="email@example.com"
+            autocomplete="off"
+            required
+          >
+        </label>
+
+        <label for="password" class="fieldsetField">
+          Password:
+
+          <input
+            type="password"
+            name="password"
+            id="password"
+            autocomplete="off"
+            minlength="5"
+            required
+          >
+        </label>
+
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>
+          An interesting fact about you!
+        </legend>
+
+        <label for="loveCatsYes" class="fieldsetField">
+          Do you love cats?
+
+          <input
+            type="radio"
+            name="loveCats"
+            id="loveCatsYes"
+            value="yes"
+          >
+          Yes
+
+          <input
+            type="radio"
+            name="loveCats"
+            id="loveCatsNo"
+            value="no"
+          >
+          No
+        </label>
+
+        <label for="favoriteColor" class="fieldsetField">
+          What is your favorite color?
+
+          <input
+            type="color"
+            name="favoriteColor"
+            id="favoriteColor"
+            value="#000000"
+          >
+        </label>
+
+        <label for="timeToGoToBed" class="fieldsetField">
+          What time do you go to bed?
+
+          <input
+            type="time"
+            name="timeToGoToBed"
+            id="timeToGoToBed"
+          >
+        </label>
+
+        <label for="favoriteCarBrand" class="fieldsetField">
+          What are your favorite brand of cars?
+
+          <select
+            name="favoriteCarBrand"
+            id="favoriteCarBrand"
+            multiple
+          >
+            <option value="BMV">
+              BMW
+            </option>
+
+            <option value="Audi">
+              Audi
+            </option>
+
+            <option value="Lada">
+              Lada
+            </option>
+          </select>
+
+        </label>
+
+        <label for="rateWork" class="fieldsetField">
+          How do you rate our work?
+
+          <input
+            type="range"
+            name="rateWork"
+            id="rateWork"
+            min="0"
+            max="10"
+            value="0"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>
+          Additional info:
+        </legend>
+
+        <label for="comments" class="fieldsetField">
+          Comments:
+
+          <textarea
+            name="comments"
+            id="comments"
+            cols="20"
+            rows="2"
+            maxlength="32"
+         ></textarea>
+        </label>
+
+        <label for="wouldRecmmemd" class="fieldsetField">
+          Would you recommend us?
+
+          <select
+            name="wouldRecmmemd"
+            id="wouldRecmmemd"
+          >
+            <option
+              value="yes"
+            >
```

Comment:
You can fit it in one line
```suggestion
            <option value="yes">
```